src/hardware/mixer.cpp

```diff
@@ -651,6 +655,35 @@ void MixerChannel::FillUp()
 	MIXER_UnlockAudioDevice();
 }
 
+const char *MixerChannel::GetLineoutMode() const
+{
+	if (left_lineout != right_lineout)
+		return right_lineout ? "Stereo" : "Reverse";
+	else
+		return right_lineout ? "RightMono" : "LeftMono";
+}
+
+bool MixerChannel::SetLineoutMode(std::string lineout_mode)
+{
+	lowcase(lineout_mode);
+	if (lineout_mode == "stereo") {
+		left_lineout = STEREO_LEFT_INDEX;
+		right_lineout = STEREO_RIGHT_INDEX;
+	} else if (lineout_mode == "reverse") {
+		left_lineout = STEREO_RIGHT_INDEX;
+		right_lineout = STEREO_LEFT_INDEX;
+	} else if (lineout_mode == "leftmono") {
+		left_lineout = STEREO_LEFT_INDEX;
+		right_lineout = STEREO_LEFT_INDEX;
+	} else if (lineout_mode == "rightmono") {
+		left_lineout = STEREO_RIGHT_INDEX;
+		right_lineout = STEREO_RIGHT_INDEX;
+	} else {
+		return false;
+	}
+	return true;
+}
```

Comment:
How about another private member function like `SetLineouts(uint8_t left, uint8_t right)` to set the lineout variables? Not a biggie, but would reduce duplication a bit.

(or `SetLineouts(uint8_t right, uint8_t left)` if you're an audiologist, because audiologists are weird like that, judging by my last audiogram)

Reply:
Good news is the enum type allowed for the creation of constexpr structs for each of the lineout modes:

![2022-03-24_17-13](https://user-images.githubusercontent.com/1557255/160030150-cc5cf5e2-c27a-416b-9258-768abc291cef.png)

Which Allowed for one-time (and exact) assignment:

![2022-03-24_17-14](https://user-images.githubusercontent.com/1557255/160030217-48d1cf80-533c-4b9b-8a9d-319c06750493.png)